src/txmempool.cpp

```diff
@@ -99,11 +100,11 @@ CTxMemPoolEntry::CTxMemPoolEntry(const CTransactionRef& tx, CAmount fee,
       nModFeesWithAncestors{nFee},
       nSigOpCostWithAncestors{sigOpCost} {}
 
-void CTxMemPoolEntry::UpdateFeeDelta(int64_t newFeeDelta)
+void CTxMemPoolEntry::UpdateModifyFee(CAmount modify_fee)
```

Comment:
I find the name `UpdateModifyFee()` confusing because both `Update` and `Modify` are verbs. What about `UpdateModifiedFee(CAmount delta)`? Or `diff` instead of `delta`? The member variable is called "modifi**ed**_fee". Or maybe even use `Increment` instead of `Update` in the name because we are doing `+=` (surely, incrementing with a negative number is actually decrementing).

(if you decide to change the name, the same change should be done to the name of `struct update_modify_fee`)

Reply:
Thx, changed to `UpdateModifiedFee(fee_diff)` as suggested.